Sources/Sentry/Public/SentrySpanContext.h

```diff
@@ -81,6 +81,16 @@ NS_SWIFT_NAME(SpanContext)
                        parentId:(nullable SentrySpanId *)parentId
                      andSampled:(BOOL)sampled;
 
+/**
+ * Sets a tag with given value.
+ */
+- (void)setTag:(NSString *)tag withValue:(NSString *)value;
+
+/**
+ * Removes a tag.
+ */
+- (void)unsetTag:(NSString *)tag;
```

Comment:
`h`: Let's stick to the way we have it on the Scope
```suggestion
- (void)setTagValue:(NSString *)value forKey:(NSString *)key NS_SWIFT_NAME(setTag(value:key:));

/**
 * Removes a tag.
 */
- (void)removeTagForKey:(NSString *)key NS_SWIFT_NAME(removeTag(key:));
```